relay-server/src/services/project.rs

```diff
@@ -1126,22 +1128,12 @@ impl Project {
 
             if limits.is_limited() {
                 // NB: Until Vec::extract_if is stable, we have to iterate twice.
-                let matching_buckets = buckets
+                let _matching_buckets = buckets
```

Comment:
Since iterators are lazy, isn't this a noop?

Reply:
Good catch thanks, I think I messed up the merge here and I ran `clippy --fix` to fix a bunch of unused imports, lesson learned.